src/main/solution_space/path_finder_test_system.py

```diff
@@ -130,10 +138,11 @@ def __get_all_possible_path_finders(self) -> List[IPathFinder]:
         return path_finders
 
     def __create_user_trees(self, test_input: TestInput) -> Tuple[AnonTree, ast.AST]:
-        code_info = CodeInfo(User(Profile(test_input[TEST_INPUT.AGE], test_input[TEST_INPUT.EXPERIENCE])))
+        code_info = CodeInfo(User(Profile(test_input[TEST_INPUT.AGE], test_input[TEST_INPUT.INT_EXPERIENCE])))
         code = Code.from_source(test_input[TEST_INPUT.SOURCE_CODE], rate=None, task=self._graph.task)
         serialized_code = SerializedCode(code, code_info, self._graph.graph_directory, self._graph.file_prefix)
         anon_tree = serialized_code.anon_trees[0]
+        anon_tree.find_medians()
```

Comment:
You often use find_medians after getting an anon tree. Maybe should we have a method **get_anon_tree_by_index** with bool param **to_find_medians**?

Reply:
Hmm, it's only when we create a new AnonTree for user (and it happens twice: here and in Hint class). I guess we should have a method that does all these steps:
1. creates Code from source
2. creates SerializedCode
3. gets the first anon_tree in serialized_code
5. finds anon medians

I think, that beyond these steps we don't need to get anon tree by index and call to_find_medians